workflows/offline_ml_diags/offline_ml_diags/_select.py

```diff
@@ -23,10 +23,12 @@ def plot_transect(
     xaxis: str = "lat",
     yaxis: str = "pressure",
     column_dim: str = "derivation",
+    dataset_dim: str = "dataset",
     figsize: Tuple[int, int] = (10, 4),
 ):
+    row_dim = dataset_dim if dataset_dim in data.dims else None
     facetgrid = data.plot(
-        y=yaxis, x=xaxis, yincrease=False, col=column_dim, figsize=figsize, robust=True
+        y=yaxis, x=xaxis, yincrease=False, col=column_dim, row=row_dim, figsize=figsize, robust=True
```

Comment:
Yup, I think I actually accidentally included this change from a separate smaller PR to handle the multiple dataset case, but I'll leave it here since it's a small change.